modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/helper/ObjectEntryHelper.java

```diff
@@ -73,9 +73,13 @@ public List<ObjectEntry> getObjectEntries(
 
 		ObjectDefinition objectDefinition =
 			_objectDefinitionLocalService.
-				getObjectDefinitionByExternalReferenceCode(
+				fetchObjectDefinitionByExternalReferenceCode(
 					objectDefinitionExternalReferenceCode, companyId);
 
+		if (objectDefinition == null) {
+			return null;
```

Comment:
If we return `null`, should we call this `fetchObjectEntries`?

Reply:
Mmm, ok, I will rename it, but the case would be the same... what if there is an error retrieving the user whose userId is stored in the object definition table?

Reply:
Well... I thought about this question. The point here is the Page is returned empty if the Object Entries do not exist. So, I don't know if it would make sense to have the fetchObjectEntries method because the list would be empty if they do not exist